content/en/account_management/billing/product_allotments.md

```diff
@@ -0,0 +1,157 @@
+
+In **Hour 3**, the organization was committed to 10 APM hosts and used 10. Their hourly Ingested Spans allotment was the maximum of their host commitment and host usage multiplied by the default allotment: `maximum(10,10) * 0.2054 GB = 2.054 GB / hour`. Their on-demand usage for the hour is the maximum of 0 and the difference between their billable usage and their allotted usage: `maximum(0, 2.054 – 2.054) = 0 GB`.
+
+Since the default usage aggregation function for Ingested Spans is sum, usage is summed over all hours in the month to get the total on-demand usage for the month. If this organization only had 3 hours of Ingested Spans usage over the month, their total monthly usage would be 0: `4452 + 0 + 0 = 0.446 GB`. 
+
+Additionally, the organization has a monthly commitment of 0.3 GB of Ingested Spans. Thus, their monthly on-demand usage is the maximum of 0 and the difference between their monthly usage and commitment: `maximum(0, 0.446 – 0.3) = 0.146 GB`.
+
+## Usage aggregation functions for allotments
+
+| Allotment             | Possible Parent Products                                      | Default monthly usage aggregation function | Default hourly usage aggregation function |
+|-----------------------|---------------------------------------------------------------|--------------------------------------------|-------------------------------------------|
+| Custom Metrics | Infrastructure Pro Hosts, Infrastructure Pro Plus Hosts, Infrastructure Enterprise Hosts, Internet of Things (IoT), Serverless Workload Monitoring - Functions, Serverless Workload Monitoring - Apps, Serverless Invocations, Serverless Functions  | Average | Average |
+| Ingested Custom Metrics | Infrastructure Pro Hosts, Infrastructure Pro Plus Hosts, Infrastructure Enterprise Hosts, Internet of Things (IoT), Serverless Workload Monitoring - Functions, Serverless Workload Monitoring - Apps | Average | Average |
+| Custom Events | Infrastructure Pro Hosts, Infrastructure Pro Plus Hosts, Infrastructure Enterprise Hosts | Sum | Sum |
+| CSM Enterprise Containers    | Cloud Security Management (CSM)       |      N/A     | Sum    |
+| CWS Containers      | Cloud Workload Security (CWS)              |       N/A     | Sum      |
+| Infrastructure Containers    | Infrastructure Pro Hosts, Infrastructure Pro Plus Hosts, Infrastructure Enterprise Hosts |   N/A   | Sum  |
+| Profiled Containers | APM Enterprise, Continuous Profiler    |   N/A        | Sum   |
+| Profiled Hosts        | APM Enterprise       | HWMP   | Sum     |
+| CI Indexed Spans    | CI Visibility         | Sum     | Sum   |      
+| Test Indexed Spans    | Test Visibility         | Sum   | Sum   |               
+| APM Indexed Spans | APM, APM Pro, APM Enterprise, Serverless APM, Legacy - Serverless Invocations, Legacy - Serverless Functions, Fargate Task (APM Pro), Fargate Task (APM Enterprise) | Sum | Sum |
```

Comment:
```suggestion
| APM Indexed Spans | APM, APM Pro, APM Enterprise, Serverless APM, </br> Legacy - Serverless Invocations, </br> Legacy - Serverless Functions, Fargate Task (APM Pro), </br> Fargate Task (APM Enterprise) | Sum | Sum |
```

Reply:
also maybe fiddle with not having the commas with the `</br>` , just thought for this section it would clean it up having those on new lines.